Neos.Flow/Classes/ObjectManagement/DependencyInjection/PropertyInjectionTrait.php

```diff
@@ -16,6 +16,8 @@
  */
 trait PropertyInjectionTrait
 {
+    protected $Flow_Injected_Properties = [];
```

Comment:
I think this can have a type hint!?
```suggestion
    protected array $Flow_Injected_Properties = [];
```